src/client/assets/templates/cla.html

```diff
@@ -19,14 +19,14 @@ <h4>{{titleMsg()}}</h4>
             <div class="cla-box well">
                 <div ng-bind-html="cla"></div>
             </div>
-            <label ng-class="required" ng-show="linkedItem.privacyPolicy">
+            <label ng-class="required" ng-show="linkedItem.privacyPolicy && (!hasCustomFields || user.value)">
                 <input ng-disabled="signed" ng-model="privacyPolicyAccepted" type="checkbox" /> I accept that the information I provide to sign this CLA will be maintained in accordance with project
                 owner's
                 <a class="highlighted-link" ng-href="{{linkedItem.privacyPolicy}}" target="space">privacy policy</a> *
             </label>
```

Comment:
Don't show this checkbox unless there are no custom fields OR if the user's signed in. Otherwise, they'll just have to click it again.